repl/evaluator.go

```diff
@@ -707,6 +746,41 @@ func (e *Evaluator) setOption(args []string) error {
 	return nil
 }
 
+func (e *Evaluator) loadExtensionOption(idx int, args []string) error {
+	if idx >= len(args) {
```

Comment:
Consider a helper function which tests the input argument count against the index and returns an error. You'll want to use the same function between this call and the `--container` check since it looks like there's a bug where not enough args are detected for the container, but there's still an array access on the invalid index.

If it's always the case that an option will be a pair of args, then consider running this check at the beginning of the for-loop whether the option name is supported or not.

Reply:
Done